modules/simple_forms_api/app/models/simple_forms_api/vba_40_10007.rb

```diff
@@ -48,6 +50,20 @@ def service(num, field, date)
       value.to_s # Convert nil to an empty string
     end
 
+    def find_cemetery_by_id(cemetery_id)
+      file_path = 'modules/simple_forms_api/app/json/cemeteries.json'
+      file_content = File.read(file_path)
+      cemeteries = JSON.parse(file_content)
+
+      cemetery = cemeteries['data'].find { |entry| entry['attributes']['cemetery_id'] == cemetery_id }
```

Comment:
Do we want to ensure cemeteries['data'] is an array before attempting to find within it? Also the safe navigation operator. 

Something like this:

```
cemetery = cemeteries['data']&.find do |entry|
  entry['attributes']&.dig('cemetery_id') == cemetery_id
end
```

Reply:
sure

Reply:
Updated